### ISSUE TITLE
Link to source code on manual pages of addons point to master branch

### ISSUE THREAD

**Issue by ecodiv** [OSGeo/grass-addons#601]
**Name of the addon**
All addons

**Describe the bug**
The links to the source code on the manual pages point to the master branch on Github. 

**To Reproduce**
1. Open the manual page of any of the addons on https://grass.osgeo.org/grass78/manuals/addons/
2. Click on 'source code' under the heading 'Source code'
3. The source code in the master branch on Github is openen

**Expected behavior**
The links to the source code on the manual pages point to the current release branch on Github. 

**System description (please complete the following information):**
 - n.a.

**Comment by ecodiv** [comment]
Relates to [#1837 in grass repository](https://github.com/OSGeo/grass/issues/1837)

**Comment by veroandreo** [comment]
I think the line to change is this one: https://github.com/OSGeo/grass/blob/75dd60a694198f7bca17b7e3d6387b6eefa9bb7a/utils/mkhtml.py#L82

Then, once addons grass8 banch is created, we should change it accordingly again

**Comment by wenzeslaus** [comment]
> Then, once addons grass8 banch is created, we should change it accordingly again

The grass8 was created.

**Comment by echoix** [comment]
Currently, the fix merged on September 26 seems to work, the source code link on the addon manual pages effectively points to the default branch. 

The current behavior, for a grass78 addon manual page, the source url points of a grass78 page points to the grass8 branch. For example, the link in https://grass.osgeo.org/grass78/manuals/addons/v.flexure.html points to https://github.com/OSGeo/grass-addons/tree/grass8/src/vector/v.flexure, and for grass80, https://grass.osgeo.org/grass80/manuals/addons/v.flexure.html also points to https://github.com/OSGeo/grass-addons/tree/grass8/src/vector/v.flexure. It is not exactly what is expected, to point us to the file that generated the page, but is tolerable.

However, it is different than the manual pages for the core algorithms. For example, the link in a grass78 https://grass.osgeo.org/grass78/manuals/r.proj.html points to https://github.com/OSGeo/grass/tree/releasebranch_7_8/raster/r.proj and https://grass.osgeo.org/grass80/manuals/r.proj.html points to https://github.com/OSGeo/grass/tree/releasebranch_8_0/raster/r.proj.

To sum up:

Addons:
- [❌] https://grass.osgeo.org/grass78/manuals/addons/v.flexure.html -> https://github.com/OSGeo/grass-addons/tree/grass8/src/vector/v.flexure
- [✔] https://grass.osgeo.org/grass80/manuals/addons/v.flexure.html -> https://github.com/OSGeo/grass-addons/tree/grass8/src/vector/v.flexure

Core:
- [✔] https://grass.osgeo.org/grass78/manuals/r.proj.html -> https://github.com/OSGeo/grass/tree/releasebranch_7_8/raster/r.proj 
- [✔] https://grass.osgeo.org/grass80/manuals/r.proj.html -> https://github.com/OSGeo/grass/tree/releasebranch_8_0/raster/r.proj 

If this is ok, the issue can be closed.

**Comment by tmszi** [comment]
> Addons:
> 
>     * [x] https://grass.osgeo.org/grass78/manuals/addons/v.flexure.html -> https://github.com/OSGeo/grass-addons/tree/grass8/src/vector/v.flexure

Thanks for catching this. It is bug.

**Comment by echoix** [comment]
I tried finding the solution by myself before posting, but I can't understand completely which script is called to generate the docs for addons. But in the core grasse mkhtml.py, I see that the version number comes from parsing an environnement variable. Does that mean that when the grass78 addons docs are generated, they are run in the same grass8 "context"?

**Comment by tmszi** [comment]
> I tried finding the solution by myself before posting, but I can't understand completely which script is called to generate the docs for addons. But in the core grasse mkhtml.py, I see that the version number comes from parsing an environnement variable. 

Bug related with case when addons are compiled on our osgeo lxd container server (e.g. https://grass.osgeo.org/grass78/manuals/addons/v.flexure.html). If you try install addon locally via `g.extension v.flexure`, addon manual page will have to correct source code and history URL (provided you have compiled the GRASS 7.8 version from last releasebranch_7_8 git branch).

> Does that mean that when the grass78 addons docs are generated, they are run in the same grass8 "context"?

G7/G8 add-ons are compiled separately on the osgeo lxd server container. The problem was that this [variable had hardcoded](https://github.com/OSGeo/grass-addons/blob/305ba6be59f479cb5e706de9cba48f40ef388e7c/utils/cronjobs_osgeo_lxd/compile_addons_git.sh#L10) version 8 as well for G7 add-ons.
